config/jobs/kubernetes-sigs/cluster-api-provider-openstack/cluster-api-provider-openstack-presubmits.yaml

```diff
@@ -23,3 +23,35 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: pull-cluster-api-provider-openstack-test
```

Comment:
you should utilize podutil to make new jobs now :-)

(aka add `decorate: true`, and you can save all bootstrap args, for example: https://github.com/kubernetes/test-infra/blob/d6f6b8a1abcacb348188cd2753aec0ab56954ad5/config/jobs/kubernetes/org/kubernetes-org-jobs.yaml#L14-L21)

Reply:
(looks like we'll need to update the sample config in readme)

Reply:
Thanks for the review! Will update accordingly :)